index.bs

```diff
@@ -478,13 +478,20 @@ still useful to render the cues to the user.</p>
 requirements.</p>
 
 <dl>
- <dt>User agents that support scripting</dt>
+ <dt>User agents</dt>
  <dd><p>All processing requirements in this specification apply. The user agent must also be
  conforming implementations of the IDL fragments in this specification, as described in the Web IDL
  specification. [[!WEBIDL]]</p></dd>
 
  <dt>User agents with no scripting support</dt>
- <dd><p>All processing requirements in this specification apply, except those in [[#api]].</p></dd>
+ <dd><p>User agents with no scripting support must comply to the same conformance criteria as user
+ agents, except that they are exempt from the requirements in [[#api]].</p></dd>
+
+ <dt>User agents with no CSS support</dt>
+ <dd><p>User agents with no CSS support must comply to the same conformance criteria as user agents,
```

Comment:
Works for me. Just want clarity over exactly which conformance classes apply in any given scenario.